Files/UserControls/NavigationToolbar.xaml

```diff
@@ -934,6 +934,25 @@
                             Modifiers="Control" />
                     </AppBarButton.KeyboardAccelerators>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="NavToolbarEnterCompactOverlay"
+                    x:Uid="NavigationToolbarEnterCompactOverlay"
+                    Click="NavToolbarEnterCompactOverlay_Click"
+                    Label="Enter compact overlay">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xE8A7;" />
+                    </AppBarButton.Icon>
+                </AppBarButton>
+                <AppBarButton
+                    x:Name="NavToolbarExitCompactOverlay"
+                    x:Uid="NavigationToolbarExitCompactOverlay"
+                    Click="NavToolbarEnterCompactOverlay_Click"
+                    Label="Exit compact overlay"
+                    Visibility="Collapsed">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xE944;" />
+                    </AppBarButton.Icon>
+                </AppBarButton>
```

Comment:
Can you try `&#xEE49;` and `&#xEE47;`? We might need to use some conditional XAML here since this icon is only available starting with 21h1.

Reply:
Do you happen to know which api contract that would be?

Reply:
I'm not sure, I did a quick search, but I didn't find anything. If you want, I can look into this more though.

Reply:
I think the icon might exist below 21h1 devices, I'll check though.